independent-projects/bootstrap/maven-resolver/src/main/java/io/quarkus/bootstrap/resolver/BootstrapAppModelResolver.java

```diff
@@ -281,6 +291,31 @@ public boolean visitLeave(DependencyNode node) {
                 .build();
     }
 
+    private void collectPlatformProperties(AppModel.Builder appBuilder, List<Dependency> managedDeps)
+            throws AppModelResolverException {
+        for (Dependency d : managedDeps) {
+            // assuming platform properties artifacts have the same reserved groupId
```

Comment:
where is this assumption in the code?

Reply:
ah, it's a leftover, i'll remove the comment.